main.go

```diff
@@ -89,6 +93,15 @@ var (
 			Usage:     "The Slack notification output template, in Golang text/template format",
 			Value:     &config.slackDescriptionTemplate,
 		},
+		{
+			Path:      alertCritical,
+			Env:       "SLACK_ALERT_CRITICAL",
```

Comment:
Should be SLACK_ALERT_ON_CRITICAL as mentioned above.

Reply:
I have updated this in the latest push.  Thanks!